src/sound/Sound.js

```diff
@@ -1233,7 +1240,7 @@ Object.defineProperty(Phaser.Sound.prototype, "volume", {
         }
         else if (this.usingAudioTag && this._sound)
         {
-            this._sound.volume = value;
+            this._sound.volume = this._globalVolume * value;
```

Comment:
This version also works for me but i don't think that sound should know about who is controlling his volume. Maybe someone made a standalone sound outside os sound manager and is controlling his volume without the use of standard phaser sound manager then this version could break some stuff for him.

I think my original version is a little bit more secure in cases when someone replaced the sound manager with their own or is using sound class without sound manager.